rules/alerting.go

```diff
@@ -413,13 +413,13 @@ func (r *AlertingRule) ActiveAlerts() []*Alert {
 	return res
 }
 
-// IsAlertUnknown returns whether an alert's RuleHealth is still unkown.
-func (r *AlertingRule) IsAlertUnknown() RuleHealth {
-	var health RuleHealth
+// IsHealthUnknown returns whether an alerts health is still unkown.
```

Comment:
1. *unknown.
2. Rephrasing of the comment might be needed, not quite clear, `... returns true if health ...`?